README.md

```diff
@@ -1,3 +1,5 @@
+**[DEPRECATED]** _This repository is no longer used, as in Gutenberg we now use the original version of this package, plus fetching [a prebuilt Android artifact from a maven repo on S3](https://github.com/wordpress-mobile/react-native-libraries-publisher/blob/trunk/README.md)._
```

Comment:
Apologies, I should have mentioned it before: When looking at the resulting page, I feel that the deprecation notice is a bit small. WDYT about making it larger, perhaps [like this one](https://github.com/wordpress-mobile/react-native-aztec#notice-this-repo-is-no-longer-maintained-the-code-from-this-repo-was-moved-to-the-gutenberg-repository-and-is-now-maintained-there-if-you-would-like-to-use-a-rich-text-editor-in-your-app-please-consider-using-gutenberg-mobile)?

Reply:
I was tempted to make it bigger but wasn't sure whether it would look better or not. Let's make it similar to the example you shared 🙇 .

Reply:
I've just applied your suggestion in https://github.com/wordpress-mobile/react-native-clipboard/pull/3/commits/8dd94a92c3a09a100d1726f8ab5646f7cdf5a40f.